client/components/Index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import {Redirect} from 'react-router-dom'
+
+const Index = props => {
+  return <Redirect to="/pets" />

```

Comment:
Not sure what this component is being used for? If it's not used we can remove it.
However, for example if we want to redirect the user from the root path to "/pets", we can add that logic to the BrowserRouter by adding an additional route

```
<Route exact path="/">
  <Redirect to="/pets" />
</Route>
```